cyipopt/__init__.py

```diff
@@ -0,0 +1,47 @@
+# -*- coding: utf-8 -*-
```

Comment:
`import cyipopt` is good!

Are you deprecating `import ipopt` so both work for this release?

Reply:
Yes, `import ipopt` raises a `FutureWarning` as you've seen below.